lib/fireblocks/client_test.go

```diff
@@ -149,20 +101,66 @@ func expectCompletedTransaction(id string) *fireblocks.TransactionResponse {
 	}
 }
 
-func parseKey(s string) *rsa.PrivateKey {
-	p, _ := pem.Decode([]byte(s))
-	if p.Type == "PRIVATE KEY" {
-		k, err := x509.ParsePKCS8PrivateKey(p.Bytes)
-		if err != nil {
-			panic(err)
-		}
+func parseKey(t *testing.T, s string) *rsa.PrivateKey {
+	t.Helper()
 
-		return k.(*rsa.PrivateKey) //nolint:forcetypeassert // parseKey is only used for testing
-	}
+	p, _ := pem.Decode([]byte(s))
```

Comment:
Maybe check error and panic/require.NoError.

Reply:
this isn't an error :)